src/main/java/com/znsio/rpi/properties/Config.java

```diff
@@ -0,0 +1,43 @@
+package com.znsio.rpi.properties;
+
+import org.apache.commons.lang3.exception.ExceptionUtils;
+import org.apache.log4j.Logger;
+
+import javax.validation.constraints.NotNull;
+import java.io.File;
+import java.io.FileInputStream;
+import java.io.IOException;
+import java.io.InputStream;
+import java.util.Properties;
+
+public class Config {
+    public static final String TARGET_ENVIRONMENT = "TARGET_ENVIRONMENT";
+    public static final String PLATFORM = "PLATFORM";
+    public static final String APP_NAME = "APP_NAME";
+    public static final String IS_VISUAL = "IS_VISUAL";
+    public static final String IS_MOBILAB = "IS_MOBILAB";
```

Comment:
Mobilab is pCloudy's on-prem installation at Jio. Not relevant here. May need to be renamed

Reply:
How about IS_DEVICEFARM?

Reply:
IS_RUNNING_ON_LOCAL ??

Reply:
Changed it to IS_LOCAL_DEVICE. And also renamed the property key from MobilabEnabled to LocalDeviceExecution